tests/ManageCourses.Tests/SmokeTests/DataServiceExportTests.cs

```diff
@@ -0,0 +1,162 @@
+        }
+
+        [OneTimeTearDown]
+        public void OneTimeTearDown()
+        {
+            if (localWebHost != null)
+            {
+                localWebHost.Dispose();
+            }
+        }
+
+
+        [Test]        
+        [Category("Smoke")]
+        [Explicit]
+        public void DataExportWithEmptyCampus()
+        {
+            var dfeSignInConfig = config.GetSection("credentials").GetSection("dfesignin");
+                                
+            var communicator = new DfeSignInCommunicator(dfeSignInConfig["host"], dfeSignInConfig["redirect_host"], dfeSignInConfig["clientid"], dfeSignInConfig["clientsecret"]);
+            var accessToken =  communicator.GetAccessTokenAsync(dfeSignInConfig["username"], dfeSignInConfig["password"]).Await();
+                            
+            var client = new ManageCoursesApiClient(new MockApiClientConfiguration(accessToken));
+            client.BaseUrl = localWebHost.Address;             
+
+            client.Data_ImportAsync(TestData.MakeSimplePayload(dfeSignInConfig["username"])).Await();
+            var export = client.Data_ExportByOrganisationAsync("ABC").Await();
+
+            Assert.IsNotNull(export, "Test could not complete");
+
+            Assert.AreEqual("123", export.OrganisationId, "OrganisationId should be retrieved");
+            Assert.AreEqual("Joe's school @ UCAS", export.OrganisationName, "OrganisationName should be retrieved");
+            Assert.AreEqual("ABC", export.UcasCode, "UcasCode should be retrieved");
+
+            Assert.AreEqual(1, export.ProviderCourses.Count, "Expecting only one in ProviderCourses");
+            var course = export.ProviderCourses.Single();
+            Assert.AreEqual(1, course.CourseDetails.Count, "Expecting only one in ProviderCourses.CourseDetails");
+            var details = course.CourseDetails.Single();
+
+            Assert.AreEqual("Joe's course for Primary teachers", details.CourseTitle, "ProviderCourses.CourseDetails.CourseTitle should be retrieved");
+                        
+            Assert.AreEqual(1, details.Variants.Count, "Expecting only one in ProviderCourses.CourseDetails.Variants");
+            var variant = details.Variants.Single();
+
+            Assert.AreEqual("XYZ", variant.UcasCode, "ProviderCourses.CourseDetails.Variants.UcasCode should be retrieved");
+
+            Assert.AreEqual(1, variant.Campuses.Count, "Expecting only one in ProviderCourses.CourseDetails.Variants.Campuses");
+            var campus = variant.Campuses.Single();
+
+            Assert.AreEqual("", campus.Code, "ProviderCourses.CourseDetails.Variants.Campuses.Code should be retrieved");                
+            Assert.AreEqual("Main campus site", campus.Name, "ProviderCourses.CourseDetails.Variants.Campuses.Name should be retrieved");
+        }
+
+        private class MockApiClientConfiguration : IManageCoursesApiClientConfiguration
```

Comment:
`DfeSignInCommunicator` wrapped by `MockApiClientConfiguration` inherits `IManageCoursesApiClientConfiguration`

pointless

Have `DfeSignInCommunicator` inherit from `IManageCoursesApiClientConfiguration` and be done with it

Reply:
But I need to jump through this hoop to use existing injection points